clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/config/KubernetesConfigurationProperties.java

```diff
@@ -55,6 +55,8 @@
     private List<String> omitKinds = new ArrayList<>();
     private boolean onlySpinnakerManaged = false;
     private Long cacheIntervalSeconds;
+    private Integer runJobPodCollectionLimit = 5;
+    private Integer runJobPodStatusCollectionLimit = 10;
```

Comment:
If these can never be null (which appears to be the case), they should be `int`s.  (If they ever were null we'd get an NPE when doing comparisons such as `pods.size() <= credentials.getRunJobPodCollectionLimit()`; by choosing the correct data type we can eliminate that failure mode.)